### ISSUE TITLE
Windows Mixed Reality Eye Gaze Data Provider Not Available

### ISSUE THREAD

**Issue by wlewis55** [microsoft/MixedRealityToolkit-Unity#9891]
MRTK 2.6.1 and 2.6.2 do not contain the WindowsMixedRealityEyeGazeDataProvider. 

![image](https://user-images.githubusercontent.com/4706315/119734340-d8411a00-be48-11eb-9ee1-d7fd7e0c7973.png)

When I try the _**Try Repair**_ button, I get the foloowing pop up window.

![image](https://user-images.githubusercontent.com/4706315/119732159-d7f34f80-be45-11eb-9806-83e45f27c2be.png)

This is similar to Issue #7284 .

Here is the error thrown by Unity:

![image](https://user-images.githubusercontent.com/4706315/119733959-3e796d00-be48-11eb-8057-7e1c3ff708a5.png)

## To reproduce

1. Set up a new Unity Project & MRTK foundation/standard assets
2. Change your main profile to the DefaultHololens2ConfigurationProfile.
3. Copy & Customize profile.
2. Any changes to MRTK profiles should trigger the error. If the error is not seen, navigate to **MRTK > Input > Input Data Providers > Windows MR Eye Gaze Provider**

## Expected behavior

I'm expecting to see something like this:

![image](https://user-images.githubusercontent.com/4706315/119733481-98c5fe00-be47-11eb-9318-2fefe1026cf4.png)

## Setup

- Unity Version 2020.3.6f1
- MRTK Version 2.6.1, 2.6.2
- Windows XR Plugin 4.4.2
- XR Plugin Management 4.0.5

## Target platform

- HoloLens 2

## Additional Comments

After doing some digging, I have a feeling this is a problem or change with Windows XR. If so, MRTK documentation needs to be fixed to reflect this. All MRTK documentation currently says something like:

- Create a new data provider:
Under Type select 'Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input' -> 'WindowsMixedRealityEyeGazeDataProvider'

as seen here: https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/input/eye-tracking/eye-tracking-basic-setup?view=mrtkunity-2021-01

**Comment by wlewis55** [comment]
Update:

The WindowsMixedRealityEyeGazeDataProvider is in the _WindowsMixedReality / XR2018_ folder within the MRTK Foundation package, but **not** in the _WindowsMixedReality / XRSDK_ package.

![image](https://user-images.githubusercontent.com/4706315/119843537-a5913300-bed5-11eb-9fa9-f74df0bada02.png)

**Comment by andikay** [comment]
I ran into the same issue while doing the Microsoft HoloLens2 MRTK tutorials with the following configuration:
* Unity 2020.1.9f1
* MRTK 2.5.4
* Windows XR Plugin 3.3.1
* XR Plugin Management 3.2.16

**Comment by MaxWang-MS** [comment]
This issue is a duplicate of #7284. Eye tracking support via Windows XR plugin is added in MRTK 2.7 (see [release notes](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/release-notes/mrtk-27-release-notes?view=mrtkunity-2021-05#support-for-eye-tracking-on-windows-xr-plugin)). @andikay Please note we only officially support the LTS versions of Unity (e.g. 2020.3). Also your WinXR plugin version is too old (see release notes). Please let us know if you encounter any issue using it in MRTK 2.7.

**Comment by MaxWang-MS** [comment]
@keveleigh Could you double check whether [the docs](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/input/eye-tracking/eye-tracking-basic-setup) is still up to date? Thanks!

**Comment by andikay** [comment]
@MaxWang-MS the MS tutorials use Unity 2019.3 LTS which is very similar to 2020.1 and MRTK 2.5.3 . I tried installing 2.6.x and the MS Feature Tool just crashes when I attempt to do that. I was able to complete all toturials with my setup except for the issue mentioned here. The suggestion to use MRTK 2.7 seems very new, when I worked through the tutorials a few days ago the MRTK 2.7 suggestion was not there and the pictures also showed MRTK 2.5.3

The workaround that works flawlessly for me is to just copy the WindowsMixedRealityEyeGazeDataProvider.cs file from the XR2018 Folder to a self-created folder in the Assets section. The file can then be found by the MRTK and can also be selected in the picture shown above by wlewis55.
